docs/tutorials/notebooks/ott-jax-tutorial-Domain-Adaptation-Romain-Louvet.ipynb

```diff
@@ -0,0 +1,1040 @@
+{
```

Comment:
<p>"the transport can be used ... using the OT plan."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ott-jax/ott/pull/364/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>